Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/rating/RatingDialog.kt

```diff
@@ -30,13 +34,18 @@ class RatingDialog : DialogFragment() {
         initializeViewModel()
         findViews(view)
         setupClickListeners()
+        loadPreviousFeedback()
 
         return view
     }
 
     private fun initializeViewModel() {
         viewModel = ViewModelProviders.of(this).get(RatingViewModel::class.java)
 
+        val userId = AuthController.userId
+        val ratingRepo = RatingRepo(userId!!, FirebaseHelper.instance.userDatabase)
```

Comment:
Another spot where we can do `userId.orEmpty()` to pass in a value without risking crashing.

Reply:
In this case, crashing is the right solution...there's no way you should be able to get to this dialog w/o being logged in....

Reply:
Gotcha. Maybe a post conference enhancement to avoid crashing but handle a different way. Not gonna trip up over it here.